cli/src/plugins/javadoc/index.ts

```diff
@@ -152,6 +153,27 @@ async function processJson(
   await Promise.all(promises);
 }
 
+function processParam(param: AnyType): string {
+  if (param.simpleTypeName.includes("Class")) {
+    return "Class";
+  } else if (param.simpleTypeName.includes("ImportFlag")) {
+    return "ImportFlag...";
+  } else if (param.simpleTypeName.includes("Iterable")) {
+    return "Iterable";
+  } else if (
+    param.simpleTypeName ===
+    "E" /*&& TODO: Figure out how to filter for E implements RealmModel a bit... better
+    (param.bounds as AnyDoc[]).filter(
+      (doc) => doc.simpleTypeName != "RealmModel"
+    ).length > 1*/
+  ) {
+    return "RealmModel";
+  } else if (param.simpleTypeName == "Callback") {
+    return "App.Callback";
+  }
+  return param.simpleTypeName;
+}
```

Comment:
A collection of nasty edge case parameter names where the ref didn't seem to match up to *any* variation of simple or qualified type names consistently.

Reply:
Why not include these comments as inline comments in the code?